### ISSUE TITLE
JSR 309 driver fails to release media session of conferences

### ISSUE THREAD

**Issue by hrosa** [Mobicents/RestComm#761]
While performing load tests of conference scenario where every caller is placed into his own conference room, I noticed that the Dialogic JSR 309 driver occasionally fails to release the media session of the conference.

Some of the commons errors show in logs are show below. The log (with warn/error thresholds) of the test, that included 15k calls, can be found [here](https://github.com/Mobicents/RestComm/files/77886/xms-ms-release-issue.txt).
```
16:40:59,620 ERROR [com.vendor.dialogic.javax.media.mscontrol.DlgcSync2AsyncMonitor] (RestComm-akka.actor.default-dispatcher-14) SYNC_2_ASYNC DlgcSync2AsyncMonitor::waitForRequestCompletion():: test - out of wait with status: Error Timeout Error Executing What: DlgcProxyHelper:unjoin Request

16:41:04,626 ERROR [com.vendor.dialogic.javax.media.mscontrol.DlgcSync2AsyncMonitor] (RestComm-akka.actor.default-dispatcher-14) SYNC_2_ASYNC DlgcSync2AsyncMonitor::waitForRequestCompletion():: test - out of wait with status: Error Timeout Error Executing What: Media Session:Release Request

16:41:26,657 ERROR [com.vendor.dialogic.javax.media.mscontrol.sip.DlgcSipB2BUA] (MSS-Executor-Thread-10) Received 408 timeout
```
Failing to release these session means that ghost signaling sessions are kept in the Media Server, as shown in the screenshot below:

<img width="611" alt="screen shot 2016-01-05 at 00 35 05" src="https://cloud.githubusercontent.com/assets/7796608/12104653/75a99886-b345-11e5-85d0-e0680ee90c0a.png">

**Comment by RomanDS** [comment]
Dialogic JSR 309 will change from ERROR - timeout to an Exception for better error notification.  indicating that unjoin from conference was requested

**Comment by hrosa** [comment]
Should be solved by #767
